api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -249,27 +219,47 @@ private PatientNameSearch getPatientNameSearch() {
 			        + "a bean implementing PatientNameSearch");
 		}
 	}
-	
-	/**
-	 * @see org.openmrs.api.GlobalPropertyListener#supportsPropertyName(java.lang.String)
-	 */
-	@Override
-	public boolean supportsPropertyName(String gpName) {
-		return RegistrationCoreConstants.GP_IDENTIFIER_SOURCE_ID.equals(gpName);
-	}
-	
+
 	@Override
 	public List<PatientAndMatchQuality> findFastSimilarPatients(Patient patient, Map<String, Object> otherDataPoints,
 	                                                            Double cutoff, Integer maxResults) {
-		return getFastSimilarPatientSearchAlgorithm().findSimilarPatients(patient, otherDataPoints, cutoff, maxResults);
+		List<PatientAndMatchQuality> matches = new LinkedList<PatientAndMatchQuality>();
```

Comment:
I would have used an ArrayList here, not a LinkedList. But I don't know the actual performance tradeoffs.